packages/data-provider/src/utils/buildQuery.ts

```diff
@@ -0,0 +1,123 @@
+import { SQL, and, not, or, sql } from 'drizzle-orm';
+import * as _ from 'lodash-es';
+import { BadRequest } from './errors.js';
+import {
+	ArrayFilterValue,
+	CombinationKeys,
+	CombinationOperator,
+	FilterOperator,
+	GreaterThanFilter,
+	LesserThanFilter,
+	Operator,
+	isArrayFilter,
+	isCombination,
+	isFilter,
+} from '@overture-stack/sqon-builder';
+
+const jsonbColumnName = 'data';
```

Comment:
Setting this variable with the column name on database of type JSONB. this is used to build queries

Reply:
does JSONB stand for json blob? never heard of it until now. may want to add a comment, or rename the variable to be more declarative outside the db context.

Reply:
`JSONB` stands for `JSON Binary` ref: [link](https://scalegrid.io/blog/using-jsonb-in-postgresql-how-to-effectively-store-index-json-data-in-postgresql/#:~:text=JSONB%20stands%20for%20%E2%80%9CJSON%20Binary,you%20should%20be%20using%20JSONB.)

Added a comment to this variable to be more declarative